waveform_editor/shape_editor/plasma_properties.py

```diff
@@ -0,0 +1,102 @@
+import imas
+import panel as pn
+import param
+from panel.viewable import Viewer
+
+from waveform_editor.util import EquilibriumInput
+
+
+class PlasmaPropertiesParams(param.Parameterized):
+    """Helper class containing parameters defining the plasma properties."""
+
+    ip = param.Number(default=-1.5e7)
+    r0 = param.Number(default=6.2)
+    b0 = param.Number(default=-5.3)
+    alpha = param.Number()
+    beta = param.Number()
+    gamma = param.Number()
+
+
+class PlasmaProperties(Viewer):
+    MANUAL_INPUT = "Manual"
+    EQUILIBRIUM_INPUT = "Equilibrium IDS"
+    input_mode = param.ObjectSelector(
+        default=EQUILIBRIUM_INPUT,
+        objects=[EQUILIBRIUM_INPUT, MANUAL_INPUT],
+        label="Plasma properties input mode",
+    )
+
+    input = param.ClassSelector(class_=EquilibriumInput, default=EquilibriumInput())
+    properties_params = param.ClassSelector(
+        class_=PlasmaPropertiesParams, default=PlasmaPropertiesParams()
+    )
+    has_properties = param.Boolean(doc="Whether the plasma properties are loaded.")
+
+    def __init__(self, equilibrium):
+        super().__init__()
+        self.equilibrium = equilibrium
```

Comment:
I'm not very happy to use the equilibrium IDS as our internal data container. Could we just create variables (or params) with the data that we need? That list is pretty short anyway (ip, r0, b0, psi, f_df_dpsi and dpressure_dpsi)

Then we can construct the equilibrium IDS for NICE when the user clicks Run?